modules/gapi/src/backends/common/serialization.cpp

```diff
@@ -702,6 +770,15 @@ I::IStream& ByteMemoryInStream::operator>> (char &atom) {
     atom = m_storage[m_idx++];
     return *this;
 }
+I::IStream& ByteMemoryInStream::operator>> (wchar_t &atom) {
+    check(sizeof(wchar_t));
+    uint8_t x[sizeof(wchar_t)];
+    for (std::size_t i = 0; i < sizeof(wchar_t); ++i) {
+        x[i] = static_cast<uint8_t>(m_storage[m_idx++]);
+        atom |= (x[i] << (i * 8));
```

Comment:
I believe you should reset `atom` first to `0`, otherwise the result may be invalid

Reply:
Good point, fixed